src/components/Card/Card.jsx

```diff
@@ -1,26 +1,28 @@
 import React, {Component} from "react";
+import { DetailsModal , toggleModal} from '../DetailsModal/DetailsModal.jsx';
 
 import './style.scss'
 
 const Card = (props) => {
   const {card: {companyLogo, company, jobTitle, isRejected}} = props;
 
   return (
-    <div className="card-container" style={isRejected ? {backgroundColor: "rgba(188,43,255,.4)", textAlign: "left" } : {} }>
+    <div className={isRejected ? "card-container rejected-cards" : "card-container"} >
```

Comment:
className=`card-container ${isRejected ? 'rejected-cards' : ''}`

Reply:
Same thing on the other places we have this pattern.

Reply:
There are backticks after = and at the end, github deletes the character

Reply:
I need help for this one, it did not work or I could not do it in right way.